### PR TITLE
KZG core functionality

### PR DESCRIPTION
**Pull request by danielSanchezQ**
This is the easiest workig version of kzg we could came up with.
Commitment + Proof + Verification
Almost everything implemented from scratch for easier understanding.
No reed solomon involved which will be introduced later on in a different PR

### CODE REVIEW

**Comment by megonen** [comment]
Maybe adding "verify setup" can be helpful to check the integrity of trusted setup procedure.

**Comment by megonen** [comment]
LGTM

File: da/test_kzg.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+from da.kzg_rs import kzg
+from da.kzg_rs.common import BYTES_PER_FIELD_ELEMENT, GLOBAL_PARAMETERS, ROOTS_OF_UNITY, GLOBAL_PARAMETERS_G2
+from da.kzg_rs.trusted_setup import verify_setup
+
+
+class TestKZG(TestCase):
+
+    @staticmethod
+    def rand_bytes(size=1024):
```
Comment (youngjoon-lee) [inline da/test_kzg.py:15]: I'm late to review this PR, but have a quick question. It seems like this parameter `size` should be `n_chunks` if I understand correctly?

Comment (danielSanchezQ) [reply da/test_kzg.py:15]: yes it is

**Pull request merged**